src/ripple/nodestore/tests/import_test.cpp

```diff
@@ -562,7 +562,7 @@ class import_test : public beast::unit_test::suite
     }
 };
 
-BEAST_DEFINE_TESTSUITE_MANUAL(import,NodeStore,ripple);
+BEAST_DEFINE_TESTSUITE(import,NodeStore,ripple);
```

Comment:
The purpose of this "test" is to allow a command line to import a rocksdb database into nudb. Marking it a non-manual test will always cause it to fail with "Missing parameter: path." While this definitely increases our coverage, its kind of pointless. Not saying we should remove it but we need to think about how we can make this "real."